checklists/daily-crm-reporting-checklist.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Daily CRM Reporting Checklist
+category: Project Management
+date: "2020-03-12"
+tags: ['worklog', 'time-management']
+description: We request the worklog from the team at the end of every day so that, we can follow our performance (man/day) in each project, and we call it "CRM". Follow the steps for the perfect CRM sharing! 
```

Comment:
```suggestion
description: We request the worklog from the team at the end of every day. So that, we can follow our performance (man/day) in each project and we call it "CRM". Follow the steps for the perfect CRM sharing! 
```